src/model/entities/ingested-media.ts

```diff
@@ -0,0 +1,6 @@
+export interface IngestedMedia {
```

Comment:
I would like to know as well.
The concept of "Ingested" is reserved for Rundown, Segment, Part and Piece and it is used so that we can modify those entities to our hearts desire during the execution of a Rundown and still be able to reset them back to their original state.
As far as I know, that isn't needed for Media?

Reply:
This is not supposed to be there, i can't believe i missed that. It will be removed.

Reply:
It has been removed!